autode/neb/original.py

```diff
@@ -100,14 +100,21 @@ def total_energy(flat_coords, images, method, n_cores, plot_energies):
     )
 
     # Run an energy + gradient evaluation in parallel across all images
-    with ProcessPool(max_workers=n_cores) as pool:
-        results = [
-            pool.submit(energy_gradient, images[i], method, n_cores_pp)
+    if isinstance(method, Method):
```

Comment:
(this is a really annoying comment, sorry)
Can we invert this so
```suggestion
    if isinstance(method, IDPP):
```
then we can get rid of the inline comment, which I'd prefer

Reply:
Removed inline comment and inverted the if check